BraveWallet/Crypto/Asset Details/AssetDetailView.swift

```diff
@@ -77,28 +77,19 @@ struct AssetDetailView: View {
       Section(
         header: WalletListHeaderView(title: Text(Strings.Wallet.transactionsTitle))
       ) {
-        if assetDetailStore.transactions.isEmpty {
+        if assetDetailStore.transactionSummarys.isEmpty {
           Text(Strings.Wallet.noTransactions)
             .font(.footnote)
         } else {
-          ForEach(assetDetailStore.transactions, id: \.id) { tx in
-            Button(action: { self.transactionDetails = tx }) {
-              TransactionView(
-                info: tx,
-                keyringStore: keyringStore,
-                networkStore: networkStore,
-                visibleTokens: [assetDetailStore.token],
-                allTokens: [], // AssetDetailView is specific to a single token
-                displayAccountCreator: true,
-                assetRatios: [assetDetailStore.token.symbol.lowercased(): assetDetailStore.assetPriceValue],
-                currencyCode: assetDetailStore.currencyCode
-              )
+          ForEach(assetDetailStore.transactionSummarys) { txSummary in
+            Button(action: { self.transactionDetails = txSummary.txInfo }) {
+              TransactionSummaryView(summary: txSummary, displayAccountCreator: true)
             }
             .contextMenu {
-              if !tx.txHash.isEmpty {
+              if !txSummary.txHash.isEmpty {
                 Button(action: {
                   if let baseURL = self.networkStore.selectedChain.blockExplorerUrls.first.map(URL.init(string:)),
-                    let url = baseURL?.appendingPathComponent("tx/\(tx.txHash)") {
+                     let url = baseURL?.appendingPathComponent("tx/\(txSummary.txHash)") {
```

Comment:
seems like introduced one more space here

Reply:
Noticed this too, same in `AccountActivityView`, but re-indent remains the same. I believe we were missing a space before.

Reply:
When we ran `swiftformat` back in the day it messed up this indentation, this is the correct Xcode indentation